### ISSUE TITLE
Customizable "Dashboard Projects.json" file location

### ISSUE THREAD

**Issue by seiks** [Kruemelkatze/vscode-dashboard#12]
:wave: Hello.

Thanks for a great extension.

Would it be possible to configure the location of `Dashboard Projects.json` file?

Currently, it gets stored in `~/tmp/Dashboard Projects.json`.

What if users could configure the path of this file with a setting? Following the established naming scheme, it could look something like this:

```json
{
    "dashboard.customProjectsJSONPath": "/customizable/path/to.json",
}
```

This option would accommodate people who keep all of their VSCode settings under version control, and people who do not want to clutter their home directory.

What do you think?

**Comment by Kruemelkatze** [comment]
Hey!
Sounds like a good idea, especially as it does not impact users that do not want to change anything.

I'll have a look on this! :)

**Comment by Kruemelkatze** [comment]
This has been implemented in version 1.3, released today. :)
